GameData/KerbalismConfig/Support/ROCapsules.cfg

```diff
@@ -601,26 +1002,47 @@
 {
 	@MODULE[Habitat]
 	{
-		%volume = 8.95
-		%surface = 35
+		%volume = 9	//m^3
+		%surface = 4.3267	//m^2
+		%max_pressure = 0.35
```

Comment:
There are many of these, but almost all of these capsules (everything after Apollo , and everything that is Soviet in general) were used at 1 atm of pressure, so saying that the max pressure these could handle is just 0.35 atm is wrong. There are many more occurrences of this, they should all be brought in line.
```suggestion
```